trunk/SUAVE/Methods/Cooling/Cryocooler/Cooling/cryocooler_model.py

```diff
@@ -0,0 +1,127 @@
+## @ingroup Methods-Cooling-Cryocooler-Cooling
+# Cryocooler_model.py
+# 
+# Created:  Nov 2019, K.Hamilton
+# Modified: Nov 2021,   S. Claridge
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+import numpy as np
+from SUAVE.Core import Units, Data
+
+# ----------------------------------------------------------------------
+#   Cryocooler Model
+# ----------------------------------------------------------------------
+
+## @ingroup Methods-Cooling-Cryocooler-Cooling
+# def cryocooler_model(max_power, cooler_type, cryo_temp, amb_temp=292.2):
+def cryocooler_model(self, cooling_power, cryo_temp, amb_temp):
```

Comment:
This is a function and not a class, it shouldn't take self. I understand that this is operates on an object, name that object and update the inputs below in the doc string.